src/frontend/src/lib/components/core/Loaders.svelte

```diff
@@ -11,11 +12,13 @@
 	<Loader>
 		<LoaderBalances>
 			<IcLoaderWallets>
-				<ExchangeWorker>
-					<LoaderMetamask>
-						<slot />
-					</LoaderMetamask>
-				</ExchangeWorker>
+				<BtcLoaderWallets>
```

Comment:
Maybe, in a future PR, we can extract a component `LoaderWallets` in which we use `<IcLoaderWallets>` and `<BtcLoaderWallets>` that way here we consume just one component?

Reply:
Sounds good, added to my todo list.